mods/coop/hook/lua/simInit.lua

```diff
@@ -3,8 +3,8 @@ function OnCreateArmyBrain(index, brain, name, nickname)
     ReallyOnCreateArmyBrain(index, brain, name, nickname)
 
     -- Stuff this army into the HumanPlayers set, if applicable.
-    if StringStartsWith(ArmyBrains[index].Name, "Player") then
-        table.insert(ScenarioInfo.HumanPlayers, index)
+    if StringStartsWith(name, "Player") then
+        ScenarioInfo.HumanPlayers[name] = index
```

Comment:
This will break other missions

Reply:
Added compatibility to maps. Coop mod is fully compatible.

https://github.com/FAForever/faf-coop-maps/pull/349
https://github.com/FAForever/NomadMissions/pull/16